### PR TITLE
feat: dashboard welcomes user by their name

### PR DESCRIPTION
**Pull request by dickeyy**
Fixes: #372 

## Proposed Changes

- Change the dashboard welcome message from "Good Afternoon, Ahmed!" to "Good Afternoon, {user}!"
- Checks if `user.firstName` exists otherwise it uses `user.emailAddresses[0].emailAddress.split("@")[0].split(".")`

Ex:
image

### CODE REVIEW

File: src/app/(dashboard)/app/page.tsx [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
   return (
     <div className="flex flex-1">
       <div className="flex-1">
         <div className="space-y-3">
-          <h1 className="text-4xl font-semibold">Good afternoon, Ahmed!</h1>
+          <h1 className="text-4xl font-semibold">
+            Good afternoon,{' '}
+            {user?.firstName
+              ? user.firstName
+              : user?.emailAddresses[0]?.emailAddress
```
Comment (Erb3) [inline src/app/(dashboard)/app/page.tsx:16]: `a ? a : b` can generally be replaced by `a || b`

File: src/app/(dashboard)/app/page.tsx [hunk 2]
```diff
@@ -1,9 +1,23 @@
+'use client';
```
Comment (ixahmedxi) [inline src/app/(dashboard)/app/page.tsx:1]: I don't think it's a good idea to change the entire page to be client rendered, I would move the greeting bit in it's own component.

Comment (dickeyy) [reply src/app/(dashboard)/app/page.tsx]: I agree, I have refactored the component!

File: src/app/(dashboard)/app/page.tsx (hunk 1 above)
Comment (dickeyy) [reply src/app/(dashboard)/app/page.tsx]: Thanks!

**Review by ixahmedxi — APPROVED** [review]
Thank you, this looks great!

**Pull request merged**